controllers/admin/AdminOrdersController.php

```diff
@@ -178,6 +178,7 @@ public function __construct()
             $order = new Order((int) Tools::getValue('id_order'));
             $this->context->cart = new Cart($order->id_cart);
             $this->context->customer = new Customer($order->id_customer);
+            $this->context->currency = new Currency($order->id_currency);
```

Comment:
This line modifies the legacy Orders controller. Does this mean you fixed the issue in both legacy and migrated pages ?

Reply:
Yes the fix is done for both controllers

Reply:
Even though it will be removed in 177 ^^
But it can be useful for people who would need to patch the legacy controller